### DISCUSSION TITLE
SCTP could not connect

### DISCUSSION THREAD

**Discussion by mouhamethdk** [aligungr/UERANSIM#711] [General]
can someone help ? i'm trying to set an automated/dockerized deployment for an 5g network with open5gs and ueransim. However, I struggle when I try to automate some commands within entrypoints. The main issue I am facing is the SCTP protocol. As of right now, if I set a number of UEs to 6 it works but with 7 I have the error: [2024-06-18 14:05:45.331] [sctp] [info] Trying to establish SCTP connection... (173.22.0.4:38412)
[2024-06-18 14:05:45.342] [sctp] [error] Connecting to 173.22.0.4:38412 failed. SCTP could not connect: Connection refused

**Comment by lakshithaHewagamage** [comment]
@mouhamethdk hey, did you get the solution, same error i am also getting.

**Comment by mouhamethdk** [comment]
Hello, yes it finally worked. I changed my docker-compose,  ran the core services first then the ran services and I did not get that error anymore. Here is a resource for sequential containers running: https://docs.docker.com/compose/profiles/

**Comment by lakshithaHewagamage** [comment]
@mouhamethdk I am not using docker, just 2 vms, 1 for core and 1 for RAN. Even I restart those functions in open5gs but still I am getting same error.

**Reply by abenezer44** [reply]
Hi i am facing the same issue. Any luck in tshooting ?
